### DISCUSSION TITLE
How to make `buffers` colored like section c, including when mode changes?

### DISCUSSION THREAD

**Discussion by arctic-penguin** [nvim-lualine/lualine.nvim#685] [Configuration]
MWE:
```vim
set nocompatible
filetype off

set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()
Plugin 'nvim-lualine/lualine.nvim'

call vundle#end()
filetype plugin indent on

lua << END
require("lualine").setup({
    sections = {
	lualine_x = {
		{'buffers',
		mode = 2,
		--buffers_color = {active = 'lualine_c_normal', inactive = 'lualine_c_inactive'}
		--buffers_color = {active = { bg = colors.darkgray, fg = colors.white }}
		}, 'encoding', 'fileformat', 'filetype'},
    },
    options = {
        theme = 'gruvbox',
    }
})
END
```

Then lualine looks like this in normal mode:
![screen08](https://user-images.githubusercontent.com/103587359/169551408-9416f778-3d5f-4a8b-9414-38328a050bab.png)

And like this in command mode:
![screen09](https://user-images.githubusercontent.com/103587359/169551446-3609bc16-c182-464e-ba94-9101d53e230f.png)

The `buffers` element is correctly colored in normal mode, but not in command mode. How can I make it so that buffers has the same color as section c, no matter what mode I am in?

**Comment by anuramat** [comment]
```lua
          {
            'buffers',
            use_mode_colors = true,
          },
```
same with tabs
UPD:
> 2022

damn
